ntp-proto/src/peer.rs

```diff
@@ -192,15 +212,10 @@ impl Peer {
             // Received answer, so no need for backoff
             self.next_poll_interval = self.last_poll_interval;
 
-            // TODO: properly fill in system parameters
-            let filter_input = FilterTuple::from_packet_default(
-                &message,
-                NtpDuration::from_seconds(0.0),
-                recv_time,
-                recv_time,
-            );
+            let filter_input =
+                FilterTuple::from_packet_default(&message, system.precision, recv_time, recv_time);
```

Comment:
just checking: the repeated `recv_time` is correct here? the spec uses `c.t` for the second one (it is the timestamp that ends up in the filtertuple).

Reply:
Note: There are issues here, these are preexisting, we will make an issue to address the second timescale this currently uses implicitly